src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -112,8 +113,13 @@ export const styles = StyleSheet.create({
   containerPhoto: {
     justifyContent: 'center',
     alignItems: 'center',
-    paddingTop: dimen.normalizeDimen(8),
-    paddingBottom: dimen.normalizeDimen(13)
+    alignSelf: 'center',
+    marginTop: dimen.normalizeDimen(8),
+    marginBottom: dimen.normalizeDimen(13),
+    width: dimen.normalizeDimen(100),
+    height: dimen.normalizeDimen(100),
+    borderRadius: dimen.normalizeDimen(100),
+    backgroundColor: COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `containerPhoto` style are good for improving the layout and positioning of the photo container. However, it's important to ensure that these changes don't negatively affect other components that might be using this style.

```diff
-    paddingTop: dimen.normalizeDimen(8),
-    paddingBottom: dimen.normalizeDimen(13)
+    alignSelf: 'center',
+    marginTop: dimen.normalizeDimen(8),
+    marginBottom: dimen.normalizeDimen(13),
+    width: dimen.normalizeDimen(100),
+    height: dimen.normalizeDimen(100),
+    borderRadius: dimen.normalizeDimen(100),
+    backgroundColor: COLORS.signed_primary
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->